.travis.yml

```diff
@@ -4,6 +4,6 @@ rvm:
 before_install:
   - |
     if [ "$LUMOS" == "true" ]; then
-      GIST="https://gist.githubusercontent.com/camertron/c3386c9533e8ed32be35/raw/c212d6b6eb6ae463310d3ae187507a46f353b2af/txgh_lumos.patch"
+      GIST="https://gist.githubusercontent.com/camertron/c3386c9533e8ed32be35/raw/b399feaa1245a7bf89d3cc04ca44a42efb2ffdb5/txgh_lumos.patch"
```

Comment:
Adding `$GIST` to the list of env vars is a great idea, I'll do that right now. I don't think it's necessary for you to switch over to the patch type of approach. I've got a few branches queued up that remove any dependence on external APIs in the tests.